crates/ev-cli/src/commands/encrypt.rs

```diff
@@ -68,7 +68,13 @@ impl CmdOutput for EncryptMessage {
 
 pub async fn run(args: EncryptArgs, auth: BasicAuth) -> Result<EncryptMessage, EncryptError> {
     let api_client = EvApiClient::new(auth);
-    let encrypted = api_client.encrypt(Value::from_str(&args.data)?).await?;
+
+    let data = match Value::from_str(&args.data) {
+        Ok(val) => val,
+        Err(_) => format!("\"{}\"", args.data).parse::<Value>()?,
```

Comment:
Easier to just create a `Value::String`?
```suggestion
        Err(_) => Value::String(args.data),
```

Reply:
nice!